resources/js/Pages/Results.vue

```diff
@@ -34,18 +57,31 @@
     interface LabelMap {
         [entityId: string]: string
     }
-
+     
     export default defineComponent({
         components: {
             Head,
             MismatchesTable,
-            WikitLink
+            WikitLink,
+            WikitButton,
+            Message
         },
         props: {
             item_ids: Array as PropType<string[]>,
             results: Object as PropType<Result>,
             labels: Object as PropType<LabelMap>
         },
+        computed: {
+            labelsFromNotFound() {
+                const notFoundItems = this.item_ids.filter( element => !Object.keys( this.results ).includes(element) );
+                
+                const notFoundLabels = Object.entries(this.labels)
```

Comment:
I don't think we should iterate over all the labels just to find the correct labels for missing items, can we just directly access the labels in the template? See suggestion below.

Reply:
Applied suggestion.